core/src/main/java/nl/nn/adapterframework/management/bus/endpoints/ConfigManagement.java

```diff
@@ -15,33 +15,309 @@
 */
 package nl.nn.adapterframework.management.bus.endpoints;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.EnumMap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.Response.Status;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Logger;
+import org.springframework.http.MediaType;
 import org.springframework.messaging.Message;
+import org.springframework.messaging.support.GenericMessage;
 
 import lombok.Getter;
 import lombok.Setter;
 import nl.nn.adapterframework.configuration.Configuration;
+import nl.nn.adapterframework.configuration.ConfigurationException;
+import nl.nn.adapterframework.configuration.ConfigurationUtils;
+import nl.nn.adapterframework.configuration.IbisContext;
 import nl.nn.adapterframework.configuration.IbisManager;
+import nl.nn.adapterframework.core.IAdapter;
+import nl.nn.adapterframework.jdbc.FixedQuerySender;
+import nl.nn.adapterframework.jndi.JndiDataSourceFactory;
+import nl.nn.adapterframework.management.bus.ActionSelector;
 import nl.nn.adapterframework.management.bus.BusAware;
 import nl.nn.adapterframework.management.bus.BusMessageUtils;
 import nl.nn.adapterframework.management.bus.BusTopic;
 import nl.nn.adapterframework.management.bus.ResponseMessage;
 import nl.nn.adapterframework.management.bus.TopicSelector;
+import nl.nn.adapterframework.receivers.Receiver;
+import nl.nn.adapterframework.util.AppConstants;
+import nl.nn.adapterframework.util.DateUtils;
+import nl.nn.adapterframework.util.LogUtil;
+import nl.nn.adapterframework.util.NameComparatorBase;
+import nl.nn.adapterframework.util.RunState;
+import nl.nn.adapterframework.webcontrol.api.ApiException;
 
 @BusAware("frank-management-bus")
+@TopicSelector(BusTopic.CONFIGURATION)
 public class ConfigManagement {
+
+	private String orderBy = AppConstants.getInstance().getProperty("iaf-api.configurations.orderby", "version").trim();
 	private @Getter @Setter IbisManager ibisManager;
+	private Logger log = LogUtil.getLogger(this);
+	private static final String HEADER_CONFIGURATION_NAME_KEY = "configuration";
+	private static final String HEADER_CONFIGURATION_VERSION_KEY = "version";
+	private static final String HEADER_DATASOURCE_NAME_KEY = "datasourceName";
+
+	private IbisContext getIbisContext() {
+		return ibisManager.getIbisContext();
+	}
 
-	@TopicSelector(BusTopic.CONFIGURATION)
+	/**
+	 * @return Configuration XML
+	 * header loaded to differentiate between the loaded and original (raw) XML.
+	 */
+	@ActionSelector("get")
```

Comment:
Kunnen de actions constanten worden, die ook in de 'frontend' zoals ShowConfiguration gebruikt kunnen worden?